src/containers/MyLists/index.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import {
+  Layout, Row, Col, Card, Typography, Modal, Pagination
+} from 'antd'
+import { PlusCircleOutlined, DeleteOutlined } from '@ant-design/icons'
+import { range } from 'lodash'
+
+import Header from '../../components/Header'
+// import CreateListModal from '../CreateListModal'
```

Comment:
can be removed

Reply:
pls, don't review components outside the Login feature